tools/jlm-hls/jlm-hls.cpp

```diff
@@ -79,6 +79,7 @@ main(int argc, char ** argv)
     jlm::hls::rvsdg2rhls(*rvsdgModule);
 
     jlm::util::filepath firrtlFile(commandLineOptions.OutputFiles_.to_str() + ".fir");
+    jlm::util::filepath outputVerilogFile(commandLineOptions.OutputFiles_.to_str() + ".v");
```

Comment:
I would move it to where it is first used (shortly before line 102), and not 20-30 lines further up. It is not used anywhere in between.

Reply:
It's as far down as possible